easybuild/easyconfigs/g/GnuTLS/GnuTLS-3.8.10-GCCcore-14.2.0.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'ConfigureMake'
+
+name = 'GnuTLS'
+version = '3.8.10'
+
+homepage = 'https://www.gnutls.org'
+description = """GnuTLS is a secure communications library implementing the SSL, TLS
+ and DTLS protocols and technologies around them. It provides a simple
+ C language application programming interface (API) to access the secure
+ communications protocols as well as APIs to parse and write X.509, PKCS #12,
+ OpenPGP and other required structures. It is aimed to be portable
+ and efficient with focus on security and interoperability."""
+
+toolchain = {'name': 'GCCcore', 'version': '14.2.0'}
+
+source_urls = ['https://www.gnupg.org/ftp/gcrypt/gnutls/v%(version_major_minor)s']
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['db7fab7cce791e7727ebbef2334301c821d79a550ec55c9ef096b610b03eb6b7']
+
+builddependencies = [
+    ('binutils', '2.42'),
+    ('pkgconf', '2.3.0'),
+]
+
+dependencies = [
+    ('GMP', '6.3.0'),
+    ('nettle', '3.10.1'),
+    ('Guile', '3.0.10'),
```

Comment:
If Guile is no longer supported, does it still need to be a dependency?

Reply:
You merged the previous one :) 

I've reworked the configopts and deps based on what i could see was used